plugins/woocommerce/client/blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.spec.ts

```diff
@@ -77,56 +107,59 @@ test.describe( 'Shopper → Notices', () => {
 	} );
 } );
 
-test.describe( 'Shopper → Translations', () => {
-	test.beforeEach( async () => {
-		await wpCLI( `site switch-language ${ translations.locale }` );
-	} );
-
-	test( 'User can see translation in empty Mini-Cart', async ( {
-		page,
-		frontendUtils,
-		miniCartUtils,
-	} ) => {
-		await frontendUtils.emptyCart();
-		await frontendUtils.goToShop();
-		await miniCartUtils.openMiniCart();
-
-		await expect(
-			page.getByRole( 'link', {
-				name: getTestTranslation( 'Start shopping' ),
-			} )
-		).toBeVisible();
-	} );
-
-	test( 'User can see translation in filled Mini-Cart', async ( {
-		page,
-		frontendUtils,
-		miniCartUtils,
-	} ) => {
-		await frontendUtils.emptyCart();
-		await frontendUtils.goToShop();
-		await frontendUtils.addToCart( SIMPLE_PHYSICAL_PRODUCT_NAME );
-		await miniCartUtils.openMiniCart();
-
-		await expect(
-			page.getByRole( 'heading', {
-				name: getTestTranslation( 'Your cart' ),
-			} )
-		).toBeVisible();
+// Skip the rest of the translation tests if the iAPI mini cart is  enabled.
```

Comment:
As the iAPI Mini-Cart evolves, do you think it would make sense to actually split the tests and treat the blocks as 2 separate blocks?

Some benefits include:
- running the legacy one without looking at the config
- easier testing approach per block; conditional tests bloat the testing
- the legacy block won't receive new features, so its tests also shouldn't change; whereas with the common tests we might have to deal long term with conditional approaches.

Reply:
Our idea was to deprecate the legacy block at some point and not support it anymore, since the new one has 100% backward compatibility and there is no case in which the old one is preferable. If that's the case, it would be enough to just delete the specific part of the tests for the legacy cart, besides deleting all the code from the old block.